src/sbml/units/UnitFormulaFormatter.cpp

```diff
@@ -377,7 +377,7 @@ UnitFormulaFormatter::getUnitDefinitionFromFunction(const ASTNode * node,
 
   if(node->getType() == AST_FUNCTION)
   {
-    const FunctionDefinition *fd = 
+    const FunctionDefinition *fd = node->getName() == NULL ? NULL :
```

Comment:
maybe add a comment here explaining why `getName()` is NULL when the function takes more than 2 arguments? Because it is not clear judging by the function name only.